### DISCUSSION TITLE
Where to ask schema usage questions?

### DISCUSSION THREAD

**Discussion by hbruch** [schemaorg/schemaorg#2792] [Q&A] [answered]
The ‚How we work‘ schema.org page does a great job describing where and how contributions to schema.org can be made.

However, as a new user, my first question is where I might get answers to my questions, e.g. how to represent various aspects I want to describe via schema. 

Is this github discussions the right place? I‘m unsure about it, because it was created only recently and not yet has many contributions.

If it is, the ‚How we work‘ page could explicitly direct new users here. If not, I‘d be thankful for a pointer to a forum/mailing list or alike.

Thx

**Comment by thadguidry** [comment, marked as answer]
Hi @hbruch You can absolutely ask questions in the Discussions here on GitHub and encourage discussion here in GitHub Discussions I would say. It helps because we can easily link to other discussions and even issues or code ! (if you really want email, we do have the mailing list public-schemaorg@w3.org)

@danbri @RichardWallis we could probably update the "How we work" page as well as the "About Schema.org" page, I would say to include this more up to date information?

@hbruch Make a new discussion with an appropriate subject and ask whatever your questions are.

**Comment by Crowbar420** [comment]
Check this out
 https://paykstrt.com/36010/115994
